src/main/java/com/moviecat/test/routers/RPCRouter.java

```diff
@@ -0,0 +1,211 @@
+    GenreServiceImpl genreService = new GenreServiceImpl();
+
+    /**
+     * Health check
+     * @return Boolean
+     */
+    @JsonRpcMethod("isAlive")
+    public boolean isAlive() {
+        return BaseService.isAlive();
+    }
+
+    /**
+     * Fetches all countries
+     * @return List<Country>
+     */
+    @JsonRpcMethod("fetchCountries")
+    public List<Country> fetchCountries() {
+        return BaseService.fetch(Country.class);
+    }
+
+    /**
+     * Fetches countries in Paginated format
+     * @param limit
+     * @param page
+     * @return List<Country>
+     */
+    @JsonRpcMethod("fetchCountriesLimit")
+    public List<Country> fetchCountriesLimit(@JsonRpcParam("limit") final int limit, @JsonRpcParam("page") final int page) {
+        return BaseService.fetch(Country.class, limit, page);
+    }
+
+    /**
+     * Creates a new Role
+     * @param id
+     * @param roles
+     * @param description
+     */
+    @JsonRpcMethod("createRole")
+    public void createRole(@JsonRpcParam("id") final String id, @JsonRpcParam("role") final String roles, @JsonRpcParam("description") final String description) {
+        Role role = new Role(id, roles, description);
+        BaseService.save(role);
+    }
+
+    /**
+     * Counts the number Roles that exist
+     * @param item
+     * @return Long
+     */
+    @JsonRpcMethod("count")
+    public Long count(@JsonRpcParam("item") final String item) {
+        Class clazz;
+        try {
+            clazz = Class.forName("com.moviecat.model." + item);
+            return BaseService.count(clazz);
+        } catch (ClassNotFoundException ex) {
+            return -1L;
+        }
+    }
+
+    /**
+     * Fetches a Role given its ID
+     * @param id
+     * @return Role
+     */
+    @JsonRpcMethod("fetchRole")
+    public Role fetchRole(@JsonRpcParam("id") final String id) {
+        return BaseService.fetchOne(Role.class, id);
+    }
+
+    /**
+     * Fetches all the Roles from data store
+     * @return List<Role>
+     */
+    @JsonRpcMethod("fetchRoles")
+    public List<Role> fetchRoles() {
+        return BaseService.fetch(Role.class);
+    }
+
+    /**
+     * Fetches Roles in Paginated Format
+     * @param limit
+     * @param page
+     * @return List<Role>
+     */
+    @JsonRpcMethod("fetchRolesLimit")
+    public List<Role> fetchRolesLimit(@JsonRpcParam("limit") final int limit, @JsonRpcParam("page") final int page) {
+        return BaseService.fetch(Role.class, limit, page);
+    }
+
+    /**
+     * Updates or edits a users Role
+     * @param id
+     * @param roles
+     * @param description
+     */
+    @JsonRpcMethod("updateRole")
+    public void updateRole(@JsonRpcParam("id") final String id, @JsonRpcParam("role") final String roles, @JsonRpcParam("description") final String description) {
+        Query<Role> query = DbProvider.connection().createQuery(Role.class).field("id=").equal(id);
```

Comment:
Update this line. Remove the `'='` to read:

`Query<Role> query = DbProvider.connection().createQuery(Role.class).field("id").equal(id);`

Reply:
Change:
Query<Role> query = DbProvider.connection().createQuery(Role.class).field("id=").equal(id);
to
Query<Role> query = DbProvider.connection().createQuery(Role.class).field("id").equal(id);